src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -39,11 +40,35 @@ class NotebookCard extends Component {
     const tenMinutesAgo = _.tap(d => d.setMinutes(d.getMinutes() - 10), new Date())
     const isRecent = new Date(updated) > tenMinutesAgo
     const notebookLink = Nav.getLink('workspace-notebook-launch', { namespace, name: wsName, notebookName: name.slice(10) })
+    const readOnlyParam = { 'read-only': 'true' }
+    const notebookReadOnlyLink = `${notebookLink}/?${qs.stringify(readOnlyParam)}`
 
     const notebookMenu = h(PopupTrigger, {
       side: 'right',
       closeOnClick: true,
       content: h(Fragment, [
+        h(MenuButton, {
+          as: 'a',
+          href: notebookLink,
+          disabled: !canWrite,
+          tooltip: !canWrite && noWrite,
+          tooltipSide: 'left'
+        }, [menuIcon('edit'), 'Open']),
+        h(MenuButton, {
+          as: 'a',
+          href: notebookReadOnlyLink,
+          tooltip: canWrite && 'Open without runtime',
```

Comment:
Maybe "doesn't use runtime"?

Reply:
I consulted with Comms and we decided to leave it as is.